system/Router/RouteCollection.php

```diff
@@ -1115,12 +1116,6 @@ public function reverseRoute(string $search, ...$params)
 			{
 				$from = key($route['route']);
 				$to   = $route['route'][$from];
-				
-				// ignore closures
-				if (! is_string($to))
```

Comment:
Is this not needed? That was recently added by another user to prevent errors. Are you seeing something different?

Reply:
I added that! It is needed because otherwise a missing reverseRoute will cause an exception. Check the Travis log for what I'm referring to on the test failures:

https://travis-ci.org/codeigniter4/CodeIgniter4/jobs/533482026

Reply:
Ohhh I see what you mean, sorry, yes I guess I didn't have that merged. Will fix.

Reply:
Re-added the tea for closures.